app/grants/views.py

```diff
@@ -375,7 +366,7 @@ def subscription_cancel(request, grant_id, grant_slug, subscription_id):
         return TemplateResponse(request, 'grants/shared/error.html', params)
 
     if request.method == 'POST' and (
-        profile == subscription.contributor_profile or request.user.has_perm('grants.change_subscription')
+        profile == subscription.contributor_profile
```

Comment:
This means the user has *full* permissions to modify subscriptions.  Do we want to remove this?

Reply:
Yeah could we test run it with permissions and then release this as a subsequent PR.
Would that be cool @owocki  ?